includes/polling/os/stoneos.inc.php

```diff
@@ -0,0 +1,4 @@
+<?php
+$hardware = $device['sysDescr'];
+$version = snmp_get($device, 'HILLSTONE-SYSTEM-MIB::sysSoftware.0', '-Ovq', 'HILLSTONE-SYSTEM-MIB');
```

Comment:
Please switch to using `snmp_get_multi_oid` instead of separate `snmp_get` calls. Remove `HILLSTONE-SYSTEM-MIB::` as you are already passing it into the snmp_get.